Terasology/LightAndShadow
# feat: re-position dialogue box and response button

Part of #132 

## Result
The dialog gets Overriden.

## Changes
* Box was moved at the bottom left
* Answer option was moved right next to the box

### Issues
* Text inside the box doesn't go along with the width of the box. E.g.: if box width gets decreased, the text inside gets hidden from the sides.

## How to test
1. Check https://github.com/Terasology/LightAndShadowResources/pull/45
2. Start a LaS game
3. Talk to The Fool NPC feature in front of you

## Test UI screenshot

<img width="694" alt="Screenshot_1" src="https://user-images.githubusercontent.com/48293545/85976420-3409e580-b9e3-11ea-811e-e90d3f0ad80e.png">
